doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -371,6 +380,117 @@ See the following table for a mapping between the devicetree input used by ``nrf
+  The CRV must be at least 15 (0xF) to ensure proper watchdog operation.
+
+UICR.SECURESTORAGE
+==================
+
+UICR.SECURESTORAGE configures the secure storage system used by |ISE| for persistent storage of cryptographic keys and trusted data.
+The secure storage is divided into separate partitions for different services and processor domains.
+The sum of all size configurations specified in ``UICR.SECURESTORAGE.*`` must be 4kB aligned.
+For more information, see :ref:`ug_nrf54h20_ironside_se_secure_storage`.
+
+The UICR.SECURESTORAGE configuration consists of the following sub-registers:
+
+UICR.SECURESTORAGE.ENABLE
+  Controls whether the secure storage feature is enabled.
+
+UICR.SECURESTORAGE.ADDRESS
+  Specifies the start address of the secure storage region in memory.
+  This address must be 4kB aligned and point to a valid memory region that can be used for secure storage.
+
+UICR.SECURESTORAGE.CRYPTO
+  Configures partition sizes for the Cryptographic service within the secure storage.
+
+  UICR.SECURESTORAGE.CRYPTO.APPLICATIONSIZE1KB
+    Sets the size of the APPLICATION domain partition for cryptographic storage, specified in 1 kiB blocks.
+
+  UICR.SECURESTORAGE.CRYPTO.RADIOCORESIZE1KB
+    Sets the size of the RADIOCORE domain partition for cryptographic storage, specified in 1 kiB blocks.
+
+UICR.SECURESTORAGE.ITS
+  Configures partition sizes for the Internal Trusted Storage (ITS) service within the secure storage.
+
+  UICR.SECURESTORAGE.ITS.APPLICATIONSIZE1KB
+    Sets the size of the APPLICATION domain partition for ITS, specified in 1 kiB blocks.
+
+  UICR.SECURESTORAGE.ITS.RADIOCORESIZE1KB
+    Sets the size of the RADIOCORE domain partition for ITS, specified in 1 kiB blocks.
+
+UICR.SECONDARY
+==============
+
+UICR.SECONDARY configures the secondary firmware boot system, which allows |ISE| to boot alternative firmware in response to specific conditions or triggers.
+This feature enables recovery scenarios and dual-firmware configurations.
+
+The UICR.SECONDARY configuration consists of multiple sub-registers organized into functional groups:
+
+UICR.SECONDARY.ENABLE
+  Controls whether the secondary firmware boot feature is enabled.
+
+UICR.SECONDARY.PROCESSOR
+  Specifies which processor should be used to boot the secondary firmware.
+  Valid values are:
+
+  * ``APPLICATION`` - Boot secondary firmware on the APPLICATION processor
+  * ``RADIOCORE`` - Boot secondary firmware on the RADIOCORE processor
+
+UICR.SECONDARY.ADDRESS
+  Sets the start address of the secondary firmware.
+  This value is used as the initial value of the secure Vector Table Offset Register (VTOR) after CPU reset.
+  The address must be aligned to a 4 kiB boundary (bits [11:0] are ignored).
+
+UICR.SECONDARY.TRIGGER
+  Configures automatic triggers that cause |ISE| to boot into secondary firmware instead of the primary firmware.
+
+  UICR.SECONDARY.TRIGGER.ENABLE
+    Controls whether automatic triggers are enabled for secondary firmware boot.
```

Comment:
is this about triggers enable so that the secondary firmware can boot? or about triggers (for something) that will  run during the secondary firmware boot? I'm guessing the first. In that case:

```
Controls whether automatic triggers are enabled to boot the secondary firmware.
```